### DISCUSSION TITLE
Welcome to Python Discussions!

### DISCUSSION THREAD

**Discussion by c17hawke** [c17hawke/Python#9] [Announcements]
## 👋 Welcome!
  We can discuss topics related to this playlist here and connect with other members. I hope that you:
  * Ask questions.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded.

To get started, comment below with a short introduction and tell us about what you do with this community.

> NOTE:  Remember that this is a community we build together 💪 So also help each other if you can when I am not available.

**Comment by DivyaAkrishnA** [comment]


**Reply by c17hawke** [reply]
You need to make it public as I can't view this notebook.
